java/arcs/core/common/Referencable.kt

```diff
@@ -19,6 +19,9 @@ interface Referencable {
     /** Unique identifier of the Referencable object. */
     val id: ReferenceId
 
+    /** Sets expiration timestamp on the Referencable object. */
+    fun setExpiration(expirationTimestamp: Long): Unit = throw NotImplementedError()
```

Comment:
can this be a `var`?
```suggestion
    var expiration: Long
        get() = TODO("not implemented")
        set(value) = TODO("not implemented")
```

Reply:
done
(as a side effect i had to remove `expirationTimestamp` from ctor parameters in RawEntity, because i couldn't override a setter inside ctor, and couldn't add a non var/val parameter in data class ctor... or did i miss anything? also it's ok to set expiration separately, not in ctor in this case, because it is only actually set, when the entity is added to a Handle, so it's even better).